pkg/resources/network_policy_acceptance_test.go

```diff
@@ -92,12 +142,89 @@ resource "snowflake_network_policy" "test" {
 `, name, networkPolicyComment)
 }
 
-func networkPolicyConfig3(name string) string {
+func networkPolicyConfigNetworkRules(name string, nameRule1 string, nameRule2 string, database string, schema string) string {
+	return fmt.Sprintf(`
+resource "snowflake_network_rule" "test1" {
+	name            = "%v"
+	database        = "%v"
+	schema          = "%v"
+	comment         = "%v"
+    type            = "IPV4"
+    mode			= "INGRESS"
+	value_list      = ["192.168.0.100/24", "29.254.123.20"]
+}
+
+resource "snowflake_network_rule" "test2" {
+	name            = "%v"
+	database        = "%v"
+	schema          = "%v"
+	comment         = "%v"
+    type            = "HOST_PORT"
+    mode			= "EGRESS"
+	value_list      = ["example.com", "company.com:443"]
+}
+
+resource "snowflake_network_policy" "test" {
+	name            = "%v"
+	comment         = "%v"
+	allowed_network_rule_list = [snowflake_network_rule.test1.qualified_name]
+	blocked_network_rule_list = [snowflake_network_rule.test2.qualified_name]
+}
+`, nameRule1, database, schema, networkPolicyComment, nameRule2, database, schema, networkPolicyComment, name, networkPolicyComment)
+}
+
+func networkPolicyConfigIPsAndRules(name string, nameRule1 string, nameRule2 string, database string, schema string) string {
+	return fmt.Sprintf(`
+resource "snowflake_network_rule" "test1" {
+	name            = "%v"
+	database        = "%v"
+	schema          = "%v"
+	comment         = "%v"
+    type            = "IPV4"
+    mode			= "INGRESS"
+	value_list      = ["192.168.0.100/24", "29.254.123.20"]
+}
+
+resource "snowflake_network_rule" "test2" {
+	name            = "%v"
+	database        = "%v"
+	schema          = "%v"
+	comment         = "%v"
+    type            = "HOST_PORT"
+    mode			= "EGRESS"
+	value_list      = ["example.com", "company.com:443"]
+}
+
+resource "snowflake_network_policy" "test" {
+	name            = "%v"
+	comment         = "%v"
+	allowed_ip_list = ["192.168.0.100/24"]
+	blocked_ip_list = ["192.168.0.101"]
+	allowed_network_rule_list = [snowflake_network_rule.test1.qualified_name]
+	blocked_network_rule_list = [snowflake_network_rule.test2.qualified_name]
+}
+`, nameRule1, database, schema, networkPolicyComment, nameRule2, database, schema, networkPolicyComment, name, networkPolicyComment)
```

Comment:
You can use https://pkg.go.dev/fmt#hdr-Explicit_argument_indexes to simplify formatting.

Reply:
Didn't know this. Thanks. Fixed.